### PR TITLE
Added .NET 6 support

### PR DESCRIPTION
**Pull request by eaba**
This PR depends on #100

### CODE REVIEW

**Comment by Aaronontheweb** [comment]
You might need to rebase this PR on all of the dependent PRs once they're merged.

File: build-system/azure-pipeline.template.yaml [hunk 1]
```diff
@@ -18,9 +18,9 @@ jobs:
         submodules: recursive  # set to 'true' for a single level of submodules or 'recursive' to get submodules of submodules
         persistCredentials: true
       - task: UseDotNet@2
-        displayName: 'Use .NET 5'
+        displayName: 'Use .NET 6'
         inputs:
-          version: '5.x'
+          version: '6.x'
```
Comment (Aaronontheweb) [inline build-system/azure-pipeline.template.yaml:23]: LGTM

File: src/Petabridge.App.Web/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/sdk:5.0 AS base
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS base
```
Comment (Aaronontheweb) [inline src/Petabridge.App.Web/Dockerfile:1]: LGTM

**Review by Aaronontheweb — COMMENTED** [review]
Overall looks good but we need to merge in your lower-level dependencies first.

**Comment by eaba** [comment]
> You might need to rebase this PR on all of the dependent PRs once they're merged.

Noted!

**Pull request merged**